machine_learning_hep/analysis/analyzer_jets.py

```diff
@@ -774,6 +799,16 @@ def _analyze(self, method = 'sidesub'):
                                     hproj,
                                     f'uf/h_{var}_{method}_unfolded_{mcordata}_' +
                                     f'jetpt-{jetptrange[0]}-{jetptrange[1]}_{i}.png')
+                                # Save the default unfolding iteration separately.
+                                if i == self.n_iter_unfold_sel - 1:
+                                    hproj_sel = hproj.Clone(f"{hproj.GetName()}_sel")
+                                    hproj_sel.Scale(1. / hproj_sel.Integral(), "width")
+                                    print(f"Final histogram: {var}, jet pT {jetptrange[0]:g} to {jetptrange[1]:g})")
+                                    print_histogram(hproj_sel)
```

Comment:
Why do we need to print the histogram here? I would really like to keep the log output reasonably clean.

Reply:
It was for debugging, comparing the results between different commits. Is it fine to print it at the `debug` level?